### DISCUSSION TITLE
setCreatedAtTimestamp and setUpdatedAtTimestamp are missing

### DISCUSSION THREAD

**Discussion by khawarizmus** [pkosiec/mongo-seeding#163] [Q&A] [answered]
I am getting this error when trying to use timestamp transformers functions:
```
Property 'setCreatedAtTimestamp' does not exist on type '{ replaceDocumentIdWithUnderscoreId: (collection: SeederCollection) => SeederCollection; }'.
```
for both functions. it seems that only `replaceDocumentIdWithUnderscoreId` is recognized at this point.

did you release this [PR](https://github.com/pkosiec/mongo-seeding/pull/129) yet?

**Comment by pkosiec** [comment]
Hi @gimyboya,
There is no release yet with the feature you described.
The plan for the next release is to implement this: https://github.com/pkosiec/mongo-seeding/issues/131, and then release 3.6.0. Hopefully I will have some time to do so before end of the year.

In the meantime, you may copy the transformers from the `mongo-seeding` codebase (https://github.com/pkosiec/mongo-seeding/blob/master/core/src/transformer/transformer-functions.ts) and use them.

Cheers!

**Comment by pkosiec** [comment, marked as answer]
The 3.6.0 was released some time ago, so I consider this conversation as answered 🙂  https://github.com/pkosiec/mongo-seeding/releases/tag/v3.6.0
